Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -3320,6 +3320,14 @@ - (void)_setObservedRenderingProgressEvents:(_WKRenderingProgressEvents)observed
     _page->listenForLayoutMilestones(layoutMilestones(observedRenderingProgressEvents));
 }
 
+- (void)_saveResources:(NSURL *)directory suggestedFileName:(NSString *)name completionHandler:(void (^)(BOOL))completionHandler
+{
+    THROW_IF_SUSPENDED;
```

Comment:
I think we need to call `completionHandler(false)` here if we're going to throw--unless throwing results in a crash, then we don't care.  :)